dev/devicelab/lib/framework/utils.dart

```diff
@@ -279,10 +279,10 @@ Future<Process> startProcess(
   assert(isBot != null);
   final String command = '$executable ${arguments?.join(" ") ?? ""}';
   final String finalWorkingDirectory = workingDirectory ?? cwd;
-  print('\nExecuting: $command in $finalWorkingDirectory'
-      + (environment != null ? ' with environment $environment' : ''));
   final Map<String, String> newEnvironment = Map<String, String>.from(environment ?? <String, String>{});
   newEnvironment['BOT'] = isBot ? 'true' : 'false';
+  newEnvironment['LANG'] = 'en_US.UTF-8';
+  print('\nExecuting: $command in $finalWorkingDirectory with environment $newEnvironment');
```

Comment:
Print the entire env, including `BOT` and `LANG`.  This makes it testable, and is probably just a good idea for accuracy.